application-ckeditor-ui/src/main/resources/CKEditor/EditSheet.xml

```diff
@@ -375,6 +375,12 @@ define('xwiki-ckeditor', [
         }))
       };
     }
+    // Define if the temporary attachment feature can be used.
+    if ($(element).attr('data-isTemporaryAttachmentSupported') === 'true') {
+      config['xwiki-upload'] = {
+        isTemporaryAttachmentSupported: true
+      };
```

Comment:
Isn't there a risk that other `xwiki-upload` configuration properties get overridden?

Reply:
Unless I'm missing something, but there's no xwiki-upload configuration right now.

Reply:
I agree, it could just make the life of future maintainers easier to assume that a config could exist (I'm nitpicking tbh).

Reply:
Note sure how that's even possible: the config object is defined just few lines above (https://github.com/xwiki-contrib/application-ckeditor/blob/52971c8c499bac38920343a5fb1f3bd24461da06/application-ckeditor-ui/src/main/resources/CKEditor/EditSheet.xml#L307), without any xwiki-upload yet. So if any other xwiki-upload config, they would be added afterwards.

Reply:
Indeed, I was reading the code as if it was in a plugin. My bad.